schemas/test/en/test_grand_calculated_summary_overlapping_answers.json

```diff
@@ -0,0 +1,329 @@
+                                            "source": "answers",
+                                            "identifier": "q1-a1"
+                                        },
+                                        {
+                                            "source": "answers",
+                                            "identifier": "q2-a1"
+                                        }
+                                    ]
+                                }
+                            }
+                        },
+                        {
+                            "type": "CalculatedSummary",
+                            "id": "calculated-summary-2",
+                            "title": "Total of eggs and cheese is calculated to be %(total)s. Is this correct?",
+                            "calculation": {
+                                "title": "eggs + cheese",
+                                "operation": {
+                                    "+": [
+                                        {
+                                            "source": "answers",
+                                            "identifier": "q1-a2"
+                                        },
+                                        {
+                                            "source": "answers",
+                                            "identifier": "q2-a2"
+                                        }
+                                    ]
+                                }
+                            }
+                        },
+                        {
+                            "type": "Question",
+                            "id": "block-3",
+                            "question": {
+                                "type": "General",
+                                "id": "question-3",
+                                "title": "Do you want to buy extra of anything this week?",
+                                "guidance": {
+                                    "contents": [
+                                        {
+                                            "description": "If you select the first option, all your answers so far will be reused in a new calculated summary for extra shopping. If you select the second option, only your answers for bread and cheese will be reused."
+                                        }
+                                    ]
+                                },
+                                "answers": [
+                                    {
+                                        "type": "Radio",
+                                        "id": "radio-extra",
+                                        "mandatory": true,
+                                        "options": [
+                                            {
+                                                "label": "Yes, I am going to buy two of everything",
+                                                "value": "AB"
```

Comment:
Minor, but clearer when the labels and values match in our schemas. This rule used to be enforced by validator but was removed so we could support dynamic answers, but still best practice to match where possible